PlanAndRide/PlanAndRide.Web/Controllers/RouteController.cs.orig

```diff
@@ -9,32 +9,34 @@ namespace PlanAndRide.Web.Controllers
     public class RouteController : Controller
```

Comment:
wygląda jak jakaś pozostałość po merge, jeżeli nie potrzebny to do usunięcia plik

Reply:
Zgadza się, niepotrzebny plik - usunięte